test/http_fakes.dart

```diff
@@ -155,8 +157,8 @@ class FakeHttpResponse implements HttpResponse {
   final HttpHeaders headers = FakeHttpHeaders();
   final Completer _completer = Completer();
   final List<int> _buffer = <int>[];
-  String _reasonPhrase;
-  Future _doneFuture;
+  String? _reasonPhrase;
+  late Future _doneFuture;
```

Comment:
can this be final?

Reply:
It can. Keep in mind that this was generated from the migration tool. Whatever changes you suggest, I'll add them.

Reply:
Done.